syntax = "proto2";
package dias;

// New key issued - option
message FreshDataAuthKey {
  required uint32 key_id = 1;
  required string key_value = 2;
}
